### DISCUSSION TITLE
Component blocks: Has something changed that I'm overlooking?

### DISCUSSION THREAD

**Discussion by cyanonoob** [keystonejs/keystone#8165] [Questions]
Hi all,

since the last update my custom component blocks are throwing an error preventing keystone from ever booting:

```
▲ [WARNING] "./component-blocks" should be marked as external for use with "require.resolve" [require-resolve-not-external]

    schema.ts:38:27:
      38 │     views: require.resolve('./component-blocks')
         ╵                            ~~~~~~~~~~~~~~~~~~~~
```

I've checked with the docs, but I'm not seeing why things should be going awry. Any insights?

**Comment by flexdinesh** [comment]
This was a [breaking change released in core v3](https://github.com/keystonejs/keystone/releases/tag/2022-10-19). Specific PR here: #7805.

You don’t have to use `require.resolve()` anymore. You can just write the path relative to your project root.
